qiling/os/windows/utils.py

```diff
@@ -138,3 +139,43 @@ def printf(ql, address, fmt, params_addr, name, wstring=False):
     ql.nprint(output)
     ql.stdout.write(bytes(stdout + "\n", 'utf-8'))
     return len(stdout), stdout
+
+
+def randomize_config_value(ql, key, subkey):
```

Comment:
Do you want delete this ? and build the example we talk about later on?

Reply:
Tomorrow going to build the example with gandcrab (since it checks these values).
Will think if this function is really necessary, and in case it is, where to put it.